sdk/keyvault/azure-keyvault-administration/conftest.py

```diff
@@ -2,7 +2,21 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 # ------------------------------------
+import os
 import sys
 
+import pytest
+
 if sys.version_info < (3, 5, 3):
     collect_ignore_glob = ["*_async.py"]
+
+
+@pytest.fixture(scope="class")
+def managed_hsm(request):
+    """Fixture for tests requiring a Managed HSM instance"""
+
+    playback_url = "https://managedhsm"
+    request.cls.managed_hsm = {
+        "url": os.environ.get("MANAGED_HSM_URL", playback_url),
```

Comment:
do you need to add MANAGED_HSM_URL to the [tests.yml](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/keyvault/tests.yml) file and as a secret to the pipeline?

Reply:
To run the tests live, yes. For now I don't plan to try that because the managed HSM instance is in the pre-prod cloud, making authentication complicated.